docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/index.rst

```diff
@@ -1,1213 +1,152 @@
-
-This is not relevant to the material above—using the Gaussian Copula to
-capture the correlation between risk-factor exposure will be fine. The
-problem is when we then layer at RiskEffect component into our model, to
-make the risk have a effect on the incidence or mortality rate of our
-cause model. Without loss of generality, suppose the risk effect will
-change the rate of disease.
-
-We have traditionally handled this by calculating a risk-deleted
-incidence:
-
-.. math::
-
-
-   i_0 = \left(1-\text{PAF}\right) \cdot i_{\text{GBD}}
-
-If we *just* include one risk effect, we should still be ok, even with
-correlation. But if we have two risk effects, we have previously used
-assumption
-
-.. math::
-
-
-   1 - \text{PAF}_{\text{Multiplicative}} = \left(1 - \text{PAF}_{\text{Risk1}}\right)\cdot\left(1 - \text{PAF}_{\text{Risk2}}\right).
-
-The more correlated the risk exposure, the more wrong this assumption.
-
-I propose we continue to assume that the relative risks have a “log
-linear” relationship with incidence:
-
-.. math::
-
-
-   i_{e_1, e_2} = i_0 \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2},
+Maintaining the correlation is tricky when the risk exposures for a single simulant change over time.
+In general, we have done this by sampling correlated *percentiles* at initialization and then keeping those *percentiles*
+static throughout a simulant's life.
+These percentiles, also known as propensities, are applied to different distributions as the simulant changes age group,
+which means that a simulant's risk exposure in one age group and the next are perfectly auto-correlated.
 
-where we have written :math:`e_1` and :math:`e_2` as “standardized”
-exposures, after scaling and shifting so that the TMREL is 0.0 and a one
-unit increment gives the relative increase in disease burden reported by
-GBD.
+For example, a simulant might have their propensities for height and weight sampled at birth from a
+multivariate uniform distribution with correlation.
+If the simulant receives a propensity of 0.85 (85th percentile) in height and 0.80 (80th percentile) in weight,
+they are initially assigned the height and weight values at those percentiles among newborns.
+But when they age into the next age group, they are assigned the values at those same percentiles in their new
+age group, and so on throughout their life.
+In this way, their height and weight can change over time, but the correlation introduced in the initial sampling
+of propensities remains.
 
-With the log-linear assumption, we can calculate the PAF from first
-principles as a double integral (below :math:`p_{e_1,e_2}` denotes the
-joint probability density of exposure level :math:`(e_1,e_2)` in the
-population:
+.. todo::
+  Be more exact about the properties of the propensity correlation approach.
+  I believe that it only exactly maintains the correlation coefficient with some distributional assumptions.
```

Comment:
Not strictly necessary to elaborate now, but I'm curious about exactly what approach you're referring to and why you think it might not be exact.

Reply:
I am having a hard time finding the right google search for this, but my instinct is that if the marginals are not Gaussian, the correlation coefficient in the copula and the correlation coefficient in the joint distribution might not be equal. In that case, if we use a single copula combined with different marginals in different age groups, the correlation coefficient might not be the same in every age group.

Reply:
I think this is **very** far down the list in terms of important questions about risk correlation :)